src/main/webapp/util.js

```diff
@@ -17,8 +17,8 @@ function xhrGet(url, callback, errback){
 	var xhr = new createXHR();
 	xhr.open("GET", url, true);
 	xhr.onreadystatechange = function(){
-		if(xhr.readyState === 4){
-			if(xhr.status === 200){
+		if(xhr.readyState == 4){
```

Comment:
=== is more specific in that it checks the type of the operand as well as the value.  See [this SO post](http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons).  Was this causing some issue?